tracetesting/transactions/tests/create-nonexisting-id.yaml

```diff
@@ -0,0 +1,43 @@
+type: Test
+spec:
+  id: zwWc1RV4R
+  name: Create with non-existing ID
+  trigger:
+    type: http
+    httpRequest:
+      url: "${env:TARGET_URL}/api/tests"
+      method: POST
+      headers:
+      - key: Content-Type
+        value: application/json
+      body: |
+        {
+          "id": "${env:EXAMPLE_TEST_ID}",
+          "name": "Pokemon - List - Get a Pokemon",
+          "serviceUnderTest": {
+            "triggerType": "http",
+            "triggerSettings": {
+              "http": {
+                "url": "${env:DEMO_APP_URL}/pokemon?take=20&skip=0",
+                "method": "GET",
+                "body": "",
+                "headers": [
+                  {
+                    "key": "Content-Type",
+                    "value": "application/json"
+                  }
+                ]
+              }
+            }
+          }
+        }
+  specs:
+  - selector: span[name = "Tracetest trigger"]
+    assertions:
+      - tracetest.selected_spans.count = 1
+      - tracetest.response.status = 200
+      - ${attr:tracetest.response.body | json_path '.id' } != ""
```

Comment:
Proposal: allow the assertion attribute to be parsed as an expression, so we can apply "filters" on it.

In this example, we have a `json_path` filter that returns the value that matches the passed jsonPath query. This could be thoug to be equivalent as:
```
func json_path(body string, jsonPathQuery string) string {
  // this is an imaginary function to exemplify the usage
  return jsonpath.Value(body, query)
}
```